tests/users/test_database_model.py

```diff
@@ -1,5 +1,4 @@
 import unittest
-import datetime
```

Comment:
why?

Reply:
it was imported but not used anywhere in the file............ flake8 correction